apps/ledger-live-desktop/src/renderer/drawers/OperationDetails/index.tsx

```diff
@@ -710,24 +695,27 @@ type OperationDetailsExtraProps = {
   operation: Operation;
   account: Account;
   type: OperationType;
-  extra: {
-    [key: string]: string;
-  };
 };
-const OperationDetailsExtra = ({ extra }: OperationDetailsExtraProps) => {
-  const jsx = Object.entries(extra).map(([key, value]) => {
-    if (typeof value === "object" || typeof value === "function") return null;
-    return (
-      <OpDetailsSection key={key}>
-        <OpDetailsTitle>
-          <Trans i18nKey={`operationDetails.extra.${key}`} defaults={key} />
-        </OpDetailsTitle>
-        <OpDetailsData>
-          <Ellipsis>{value}</Ellipsis>
-        </OpDetailsData>
-      </OpDetailsSection>
-    );
-  });
+const OperationDetailsExtra = ({ operation }: OperationDetailsExtraProps) => {
+  let jsx = null;
+
+  // Safety type checks
+  if (operation.extra && typeof operation.extra === "object" && !Array.isArray(operation.extra)) {
+    jsx = Object.entries(operation.extra as Object).map(([key, value]) => {
+      if (typeof value === "object" || typeof value === "function") return null;
```

Comment:
It seems weird that the `operation.extra["xxx"].value` could be a function ? Is it ever the case ?

Reply:
In practice it will never be, but because we now define it as `unknown` TS wants to make sure :)